docs/DeveloperGuide.md

```diff
@@ -1,14 +1,157 @@
+|-------------|--------------|------------------|------------------|
+|1|Enter Command: | view one shift | user can type in `help` for a list of commands|
+|2|Enter Shift date (in dd/MM/YYYY):|11/04/2021|date format is specified|
+|3|Enter Shift index | 1 | Shift index is from 1-6, each index representing a 4 hr block|
+|4|The people assigned to the shift are: [Adam,Eve]|-|Empty array will be returned if no employee is assigned for the shift|
+
+Step 1: The user launches the program, with the data already loaded. The application then prompts the user for their command.
+
+Step 2: The user inputs a date, and the date is processed by a DateTimeFormatter to ensure that it is in the correct format. 
+This date will be added into the shift ArrayList. 
+
+Step 3: The user inputs a shift index, which represents which shift the user is looking for. For instance, index 1 refers to 
+0000-0400 in the morning, which will be relevant to 24hr fast food chains such as McDonalds!
+
+
+Given below is what will be shown if wrong/invalid inputs are given
+
+|Step Number| System Output| Wrong User Input | System Output|
+|-------------|--------------|------------------|------------------|
+|1|Enter Command: | view one Shift | invalid command|
+|2|Enter Shift date (in dd/MM/YYYY):|12/05/2021|Date chosen has no shifts|
+|3|Enter Shift index | 1 | Shift index selected is not available|
+
+Given below is a sequence diagram of the View One Shift feature
+![img.png](img.png)
+
+### add schedule to employee feature
+#### Implementation
+
+The add schedule feature uses of the `Employee` class in the `employee` package to store the schedules of each employee. 
+
+The class diagram for the `Employee` class is as follows:
+
+![employee class diagram v2.PNG](employee%20class%20diagram%20v2.PNG)
+
+Each `Employee` object has an employee name, and an arrayList of schedules, together with the functionality to set/get the employee name, get all of the employee's schedules, add or drop a schedule from the employee object.
+
+The arraylist of schedules contains all of the employee's schedules. Each schedule is a String which follows certain constraints as described below:
+
+A schedule must be in the format: dd/mm/yyyy
+
+dd must be an integer between 1-31
+
+mm must be an integer between 1-12
+
+yyyy must be an integer between 2021-2099
+
+The validation for a schedule is done internally in the `Employee` Class through a method called `isScheduleValid`, which is used in the `addSchedule` method in the `employee` class. Only when `isScheduleValid` returns true, the `addSchedule` method will add the schedule to the employee.
+
+The logic for allowing the user to add a new schedule is in the `EmployeeController` class in the `employee` package and the `Employee` class in the `employee` package. The sequence diagram for the interaction is shown below:
+
+![add schedule sequence diagram v2.PNG](add%20schedule%20sequence%20diagram%20v2.PNG)
```

Comment:
<img width="962" alt="Screenshot 2021-04-02 at 1 00 21 PM" src="https://user-images.githubusercontent.com/58291519/113382151-6356e400-93b3-11eb-9d62-b1b3491f2830.png">

To add on, I realise that the style of this particular diagram is different from the other diagram. Perhaps it can be better to streamline the design with the other UML diagram.